app/controllers/application_controller.rb

```diff
@@ -62,6 +75,14 @@ def check_no_institution!
     end
   end
 
+  def after_sign_in_path_for(resource) 
```

Comment:
Please do not change the after sign in path for admins, let them access the administration menu by adding it to the top level navigation.